pyswmm/links.py

```diff
@@ -0,0 +1,693 @@
+        >>> with Simulation('../test/TestModel1_weirSetting.inp') as sim:
+        ...     link = Links(sim)["C1:C2"]
+        ...     print link.seepagerate
+        >>> 0
+        """
+        return self._model.getLinkParam(self._linkid,LinkParams.seepRate)
+
+    @seepagerate.setter
+    def seepagerate(self, param):
+        """
+        Set Link Average Seepage Loss
+
+        :param float param: New Parameter value
+  
+        Examples:
+
+        >>> from pyswmm import Simulation
+        >>>
+        >>> with Simulation('../test/TestModel1_weirSetting.inp') as sim:
+        ...     link = Links(sim)["C1:C2"]
+        ...     print link.seepagerate
+        ...     link.seepagerate = 0.2
+        ...     print link.seepagerate
+        >>> 0
+        >>> 0.2  
+        """  
+        self._model.setLinkParam(self._linkid,LinkParams.seepRate, param)
+        
+    @property
+    def flow(self):
+        """
+        Get Link Results for Flow. If Simulation is not running
+        this method will raise a warning and return 0. 
+
+        :return: Paramater Value
+        :rtype: float
+        
+        Examples:
+
+        >>> from pyswmm import Simulation
+        >>>
+        >>> with Simulation('../test/TestModel1_weirSetting.inp') as sim:
+        ...     link = Links(sim)["C1:C2"]
+        ...     for step in sim:
+        ...         print link.flow
+        >>> 0
+        >>> 1.2
+        >>> 1.5
+        >>> 1.9
+        >>> 1.2
+        """  
+        return self._model.getLinkResult(self._linkid,LinkResults.newFlow)
+    
+    @property
+    def depth(self):
+        """
+        Get Link Results for Depth. If Simulation is not running
+        this method will raise a warning and return 0. 
+
+        :return: Paramater Value
+        :rtype: float
+        
+        Examples:
+
+        >>> from pyswmm import Simulation
+        >>>
+        >>> with Simulation('../test/TestModel1_weirSetting.inp') as sim:
+        ...     link = Links(sim)["C1:C2"]
+        ...     for step in sim:
+        ...         print link.depth
+        >>> 0
+        >>> 1.2
+        >>> 1.5
+        >>> 1.9
+        >>> 1.2
+        """  
+        return self._model.getLinkResult(self._linkid,LinkResults.newDepth)
+    
+    @property
+    def volume(self):
+        """
+        Get Link Results for Volume. If Simulation is not running
+        this method will raise a warning and return 0. 
+
+        :return: Paramater Value
+        :rtype: float
+        
+        Examples:
+
+        >>> from pyswmm import Simulation
+        >>>
+        >>> with Simulation('../test/TestModel1_weirSetting.inp') as sim:
+        ...     link = Links(sim)["C1:C2"]
+        ...     for step in sim:
+        ...         print link.volume
+        >>> 0
+        >>> 1.2
+        >>> 1.5
+        >>> 1.9
+        >>> 1.2
+        """  
+        return self._model.getLinkResult(self._linkid,LinkResults.newVolume)
+    
+    @property
+    def usxsectionarea(self):
```

Comment:
`ups_xsection_area` ? we can go over what names and conventiones should we use to make them short enough but understandable enough